### DISCUSSION TITLE
Flashing AI DECK

### DISCUSSION THREAD

**Discussion by AbirahamLincoln** [bitcraze/discussions#161] [Q&A]
while flashing ai deck with 
$ cfloader flash aideck_gap8_wifi_img_streamer_with_ap.bin deck-bcAI:gap8-fw -w radio://0/80/2M/E7E7E7E7E8 command it stops at 2 percentage..

Reset to bootloader mode ...
Could not save cache, no writable directory
Could not save cache, no writable directory
Skipping bcAI:esp, not in the target list
Deck bcAI:gap8, reset to bootloader
| 0% Writing to bcAI:gap8 deck memory
/ 1% Writing to bcAI:gap8 deck memory
- 2% Writing to bcAI:gap8 deck memory
       
![Screenshot from 2022-07-27 22-14-22](https://user-images.githubusercontent.com/79906829/181304085-fc865867-d608-4261-a45c-eaaf1de39a0c.png)

**Comment by krichardsson** [comment]
This issue is likely connected to #159, let's see if it works when we have figured that one out.
